lib/linter/rubocop.rb

```diff
@@ -0,0 +1,54 @@
+require 'rubocop'
+
+module Linter
+  class Rubocop
+    def initialize(branch)
+      @branch = branch
+    end
+
+    def run
+      require 'tempfile'
+      result = Dir.mktmpdir do |dir|
+        @work_dir = File.join(dir, "rubocop")
+        Dir.mkdir(@work_dir)
+        collect_files
+
+        options = {:format => 'json'}
+        @branch.logger.info("Executing: #{AwesomeSpawn.build_command_line('rubocop', options)}")
```

Comment:
I just realized you don't have the .rubocop.yml from the repo, so what's it going to use?